velocity.js

```diff
@@ -1,14 +1,29 @@
 // TODO: license
-const express = require('express');
-const session = require('express-session');
 const bodyParser = require('body-parser');
+const config = require(`${__dirname}/Backend/config.js`);
+const express = require('express');
+const i18n = require("i18n");
 const pug = require('pug');
+const session = require('express-session');
+
 const app = express();
 
-const config = require(`${__dirname}/Backend/config.js`);
+// Setting up i18n library
+i18n.configure({
+    locales:['en'],
+    defaultLocale: 'en',
+    directory: __dirname + '/locales',
+    objectNotation: true
+});
+
+app.set('view engine', 'pug');
 
+app.use('/jquery', express.static(__dirname + '/node_modules/jquery/dist/'));
```

Comment:
Inconsistent convertion of dirname, config added it as a string through ${__dirname}

Reply:
well it's not inconsistent, it will look like that once converted into string interpolation. basically `${}` is used when you are using string interpolation in order to add js code

Reply:
To clarify, I mean that we should use string interpolation right off instead of adding strings

Reply:
yup I will be doing that, thanks for pointing it out